src/store/index.js

```diff
@@ -136,7 +137,7 @@ export default new Vuex.Store({
 	actions: {
 		async boot(context) {
 			const base_urls = {
-				'development': 'http://localhost:8080/api/v1/',
+				'development': 'https://api.cookbookshq.com/api/v1/',
```

Comment:
Please undo this change, use an environment variable instead and set in your .env file.

Reply:
Done